src/pages/settings/Subscription/CardSection/CardSection.tsx

```diff
@@ -105,6 +125,18 @@ function CardSection() {
                 </View>
 
                 {isEmptyObject(defaultCard?.accountData) && <CardSectionDataEmpty />}
+
+                {billingStatus?.isRetryAvailable !== undefined && (
+                    <Button
+                        text={translate('subscription.cardSection.retryPaymentButton')}
+                        isDisabled={isOffline || !billingStatus?.isRetryAvailable}
+                        isLoading={subscriptionRetryBillingStatusPending}
+                        onPress={handleRetryPayment}
+                        style={[styles.w100, styles.mt5]}
```

Comment:
There is a case where when the first menu item is `Request early cancellation`, there is no space between `Add payment card` and `Request early cancellation`.

Reply:
@c3024 can you provide a screenshot and repro steps? Then I will make a issue and cc you on it

Reply:
Sorry, it was already fixed. I mentioned this as part of the checklist. I forgot to link the issue here. #46180